runtime/doc/usr_21.txt

```diff
@@ -155,11 +155,11 @@ to be lost.  Each item can be remembered only once.
 
 GETTING BACK TO WHERE YOU STOPPED VIM
 
-You are halfway editing a file and it's time to leave for holidays.  You exit
-Vim and go enjoy yourselves, forgetting all about your work.  After a couple
-of weeks you start Vim, and type:
+You are halfway through editing a file and it's time to leave for holidays.
```

Comment:
I disagree on this one - the original is non-parseable English. "To be halfway editing a file" sounds like you're half editing and half cooking dinner or half editing and half asleep.